src/vt-tv/render/render.cc

```diff
@@ -146,7 +146,7 @@ std::pair<TimeType, TimeType> Render::compute_object_load_range() {
   this->object_load_max_ = oq_max;
 
   // return range
-  return std::make_tuple(oq_min, oq_max);
+  return std::make_pair(oq_min, oq_max);
```

Comment:
Are we setting the C++ standard high enough? `std::make_tuple` should be available.